libvector/libvector.h

```diff
@@ -33,8 +33,8 @@ typedef struct vector {
  * vector_destroy
  * vector_empty
  * vector_push_back
- * vector_pop_back
- * vector_back
+ * vector_pop_back  : only decrease the vector size
+ * vector_back      : pop the last item
```

Comment:
according to [http://www.cplusplus.com/reference/vector/vector](vector) 
pop_back() type is void, and removes the last element in the vector;
back() return reference to the last element in the vector.